lib/tasks/grda_warehouse.rake

```diff
@@ -294,6 +294,13 @@ namespace :grda_warehouse do
     rescue StandardError => e
       puts e.message
     end
+    begin
+      Rake::Task['driver:hmis:process_activity_logs'].invoke if ENV['ENABLE_HMIS_API'] == 'true'
```

Comment:
checking `HmisEnforcement.hmis_enabled?` will check the env variable and that the hmis driver is loaded. So you could use that for consistency.